glusterfs/0.2.1/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+glusterfs-server:
+  image: llparse/glusterfs:v0.2.1
+  cap_add:
+    - SYS_ADMIN
+  volumes_from:
+    - glusterfs-data
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.network: 'true'
+    io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/glusterfs-daemon
+    io.rancher.container.hostname_override: container_name
+    io.rancher.sidekicks: glusterfs-probe,glusterfs-data,glusterfs-volume-create
+  command: glusterd -p /var/run/gluster.pid --no-daemon --log-file=-
+glusterfs-probe:
+  image: llparse/glusterfs:v0.2.1
+  net: ${network_mode}
+  volumes_from:
+    - glusterfs-data
+  labels:
+    io.rancher.container.hostname_override: container_name
+  command: /opt/rancher/peerprobe.sh
```

Comment:
I thought we were adding a health check and giddyup was going ot be the entrypoint?